testworld/nft_test.go

```diff
@@ -6,38 +6,39 @@ import (
 	"net/http"
 	"testing"
 
+	"github.com/centrifuge/go-centrifuge/documents"
+
 	"github.com/centrifuge/go-centrifuge/config"
 	"github.com/stretchr/testify/assert"
 )
 
 func TestPaymentObligationMint_invoice_successful(t *testing.T) {
-	t.SkipNow()
 	t.Parallel()
 	tests := []struct {
 		name                                     string
 		grantAccess, tokenProof, readAccessProof bool
 	}{
-		{
```

Comment:
these are also not very useful

Reply:
can remove them instead of commenting pls

Reply:
Im keeping them to enable failing tests later